### ISSUE TITLE
rpm 4.20 declarative build: rpmlint warns about missing sections

### ISSUE THREAD

**Issue by DimStar77** [rpm-software-management/rpmlint#1311]
openSUSE Tumbleweed has just been updated to RPM 4.20 and I started testing/playing the new declarative build system

a simple package can be found at 
https://build.opensuse.org/package/show/home:dimstar:meson/libspelling

(note the `BuildSystem: meson` part, which in turn means the simplest of packages can skipt prep/conf/build/install/check phases (all done implicit by the BuildSystem expansion)

So far the theory - and in practice, this works (as one can see, the package builds successful and is usable)

RPMLint though does not know/understand this concept (yet) and warns about:

```
libspelling.spec: W: no-%prep-section
The spec file does not contain a %prep section.  Even if some packages don't
directly need it, section markers may be overridden in rpm's configuration to
provide additional 'under the hood' functionality.  Add the section, even if
empty.

libspelling.spec: W: no-%check-section
The spec file does not contain an %check section. Please check if the package
has a testsuite and what it takes to enable the testsuite as part of the
package build. If it is not possible to run it in the build environment
(OBS/koji) or no testsuite exists, then please ignore this warning. You should
not insert an empty %check section.

libspelling.spec: W: no-%build-section
The spec file does not contain a %build section.  Even if some packages don't
directly need it, section markers may be overridden in rpm's configuration to
provide additional 'under the hood' functionality, such as injection of
automatic -debuginfo subpackages.  Add the section, even if empty.
```

Technically, it's true, there are no such sections in the spec file - but as said: they don't have to be there unless one wants to override them
